### PR TITLE
[WIP] Fix appveyor and readme

### PR DESCRIPTION
**Pull request by cchderrick**
I believe it safe to drop v0.5 test status when julia v0.7 is in REQUIRE.

### CODE REVIEW

**Comment by cchderrick** [comment]
hmm... no test on x64, failed on x86. Let me see if I could fix it real quick.

File: README.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ TextAnalysis
 A Julia package for working with text.
 
 [![Build Status](https://api.travis-ci.org/JuliaText/TextAnalysis.jl.svg)](https://travis-ci.org/JuliaText/TextAnalysis.jl)
-[![TextAnalysis](http://pkg.julialang.org/badges/TextAnalysis_0.5.svg)](http://pkg.julialang.org/?pkg=TextAnalysis)
 [![TextAnalysis](http://pkg.julialang.org/badges/TextAnalysis_0.6.svg)](http://pkg.julialang.org/?pkg=TextAnalysis)
+[![TextAnalysis](http://pkg.julialang.org/badges/TextAnalysis_0.7.svg)](http://pkg.julialang.org/?pkg=TextAnalysis)
```
Comment (aviks) [inline README.md]: unfortunately, `pkg.julialang.org` does not provide badges for 0.7 onwards. Might as well remove these altogether.

Comment (cchderrick) [reply README.md]: Okay, thanks, didn't know that. Should I replace it with travis-ci and appveyor bages?

Comment (aviks) [reply README.md]: Yeah.

**Comment by cchderrick** [comment]
It seems Flux v0.6.10 failed to precompile on x86, I don't think anything we can fix in this PR. Maybe we need to lower the Flux version back to where x86 is supported?

**Comment by aviks** [comment]
Merging this. I'll take a look at the x86 issue separately. 
I think it's a problem with the way the binary model is serialised and restored

**Pull request merged**